resilience4j-circuitbreaker/src/test/java/io/github/resilience4j/circuitbreaker/CircuitBreakerTest.java

```diff
@@ -1194,6 +1195,41 @@ public void shouldDecorateFutureAndReturnWithCallNotPermittedException() {
         assertThat(metrics.getNumberOfNotPermittedCalls()).isEqualTo(1);
     }
 
+    @Test
+    public void shouldIgnoreUnknownExceptionsWhenSetToTrue() {
```

Comment:
I don't think that these tests are helpful.
The simulateApplicationStartup is not testing any internal logic of the CircuitBreaker.